app/Svg/Parser.hs

```diff
@@ -362,6 +362,7 @@ updateShape fill r =
                               Hybrid   -> Hybrid
                               BoolNode -> BoolNode
                               Node     -> Node
+                              _ -> ""
```

Comment:
@frankjohnlee This is the right idea, but contains a type error. You can use `undefined` instead of `""` here.